[2023-04-25T08:37:29.809+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: slack_integration.slack_integration_success manual__2023-04-25T08:37:28.671922+00:00 [queued]>
[2023-04-25T08:37:29.815+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: slack_integration.slack_integration_success manual__2023-04-25T08:37:28.671922+00:00 [queued]>
[2023-04-25T08:37:29.815+0000] {taskinstance.py:1288} INFO - 
--------------------------------------------------------------------------------
[2023-04-25T08:37:29.816+0000] {taskinstance.py:1289} INFO - Starting attempt 1 of 1
[2023-04-25T08:37:29.816+0000] {taskinstance.py:1290} INFO - 
--------------------------------------------------------------------------------
[2023-04-25T08:37:29.824+0000] {taskinstance.py:1309} INFO - Executing <Task(SlackWebhookOperator): slack_integration_success> on 2023-04-25 08:37:28.671922+00:00
[2023-04-25T08:37:29.828+0000] {standard_task_runner.py:55} INFO - Started process 2031 to run task
[2023-04-25T08:37:29.831+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'slack_integration', 'slack_integration_success', 'manual__2023-04-25T08:37:28.671922+00:00', '--job-id', '117', '--raw', '--subdir', 'DAGS_FOLDER/Task4.py', '--cfg-path', '/tmp/tmp0h0igt16']
[2023-04-25T08:37:29.831+0000] {standard_task_runner.py:83} INFO - Job 117: Subtask slack_integration_success
[2023-04-25T08:37:29.879+0000] {task_command.py:389} INFO - Running <TaskInstance: slack_integration.slack_integration_success manual__2023-04-25T08:37:28.671922+00:00 [running]> on host d51842477fac
[2023-04-25T08:37:29.928+0000] {taskinstance.py:1518} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=slack_integration
AIRFLOW_CTX_TASK_ID=slack_integration_success
AIRFLOW_CTX_EXECUTION_DATE=2023-04-25T08:37:28.671922+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-04-25T08:37:28.671922+00:00
[2023-04-25T08:37:29.929+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/providers/slack/operators/slack_webhook.py:173: UserWarning: Found unexpected keyword-argument(s) 'link_names' in `send` method. This argument(s) have no effect.
  link_names=self.link_names,

[2023-04-25T08:37:29.937+0000] {base.py:73} INFO - Using connection ID 'slack' for task execution.
[2023-04-25T08:37:30.427+0000] {taskinstance.py:1332} INFO - Marking task as SUCCESS. dag_id=slack_integration, task_id=slack_integration_success, execution_date=20230425T083728, start_date=20230425T083729, end_date=20230425T083730
[2023-04-25T08:37:30.457+0000] {local_task_job.py:212} INFO - Task exited with return code 0
[2023-04-25T08:37:30.475+0000] {taskinstance.py:2596} INFO - 0 downstream tasks scheduled from follow-on schedule check
